### ISSUE TITLE
Login Screen issues

### ISSUE THREAD

**Issue by ys1382** [ClearKeep/ck-ios#208]
1. Remove Facebook and Apple logins
2. CK logo is partially obscured by safe-area
3. "Advanced server settings" doesn't fit. Change to a gear icon.

<img width="565" alt="215560050-e037a351-1db1-4e6c-a08e-371e00a483ee" src="https://user-images.githubusercontent.com/1097496/216213615-9dec443b-3478-4d55-a4d1-ab493aa7c222.png">

**Comment by EddieLukeAtmey** [comment]
1. You can't remove Apple login, if you still wish to publish clearkeep on appstore.
From: https://developer.apple.com/forums/thread/124006

```
The guidelines have been update recently for this point:
4.8 Sign in with Apple
Apps that exclusively use a third-party or social login service (such as Facebook Login, Google Sign-In, Sign in with Twitter, Sign In with LinkedIn, Login with Amazon, or WeChat Login) to set up or authenticate the user’s primary account with the app must also offer Sign in with Apple as an equivalent option. A user’s primary account is the account they establish with your app for the purposes of identifying themselves, signing in, and accessing your features and associated services.
Sign in with Apple is not required if:
Your app exclusively uses your company’s own account setup and sign-in systems.
Your app is an education, enterprise, or business app that requires the user to sign in with an existing education or enterprise account.
Your app uses a government or industry-backed citizen identification system or electronic ID to authenticate users.
Your app is a client for a specific third-party service and users are required to sign in to their mail, social media, or other third-party account directly to access their content.
...
Existing apps must follow this guidelines before April 2020.
```

Do you still wish to remove Facebook?

2. The issue has been fixed in PR #206 
3. Will do.

**Comment by ys1382** [comment]
Thanks for the update. So #1 is not required - keep both iOS and FB.
